_shows/18_19/comic_potential.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Comic Potential
+playwright: Alan Ayckbourn
+period: Spring
+season: Postgrads
+season_sort: 380
+date_start: 2019-05-22
+date_end: 2019-05-25
+venue: New Theatre
+
+cast:
+- role: Jacie Triplethree
+  name: Eliana Fiorelli
+  person: false
+- role: Adam Trainsmith
+  name: Pete Iwanciw
+  person: false
+- role: Chandler Tate
+  name: Manolis Giouvanakis
+  person: false
+- role: Carla Pepperbloom
+  name: Kenza Louchene
+  person: false
+- role: Prim Spring
+  name: Ulrike Kuchner
+  person: false
+- role: Trudi Floote
+  name: Francesca Guardavaccaro / Francesca Manna
+  person: false
+- role: Doctor / Prostitute
+  name: Carmine Napoli
+  person: false
+- role: Farmer / Hotel Desk Clerk
+  name: Spyros Lappos
+  person: false
+- role: Mother / Wife
+  name: Lola Vazquez Peraita
+  person: false
+- role: Daughter
+  name: Laura Bisigello
+  person: false
+- role: Lester Trainsmith
+  name: Julian Bates
+  person: false
+- role: Marmion
+  name: Sargis Avetysian
+  person: false
+- role: Turkey
+  name: Graham Smith
+  person: false
+- role: Girl In Dress Ship
+  name: Fiorella Di Fina / Laura De Simoni
+  person: false
+- role: Man In Dress Shop
+  name: Fabrizio Di Napoli
+  person: false
+- role: Dress Shop Assistant
+  name: Laura Bisigello / Emma Garner
+  person: false
+- role: Waiter 1
+  name: Paolo Leorin / Lillo Salvaggio
+  person: false
+- role: Waiter 2
+  name: Lillo Salvaggio / Nicola Franchini
+  person: false
+- role: Love
+  name: Fabio Tanania
+  person: false
+- role: Technician
+  name: Gordon Kershaw
+  person: false
+
+crew:
+- role: Director
+  name: Nicola Franchini
+  person: false
+- role: Director
+  name: Gordon Kershaw
+  person: false
+- role: Producer
+  name: Julie Daujat
+  person: false
+- role: Producer
+  name: Leigh Beard
+  person: false
+- role: Lighting Designer
+  name: Nicola Franchini
+  person: false
+- role: Music
+  name: Pete Iwanciw
+  person: false
+- role: Music
+  name: Gordon Kershaw
+  person: false
+- role: Music
+  name: Giulia Grisot
+  person: false
+- role: Tech Box and Sound Effects
+  name: Lorenzo Bordin
+  person: false
+- role: Tech Box and Sound Effects
```

Comment:
These two are Technical Operators, with Lorenzo also a Lighting Designer. 
Zoe was also a Lighting Designer, and I wouldn’t say no to a Venue Technician role 😛